src/app/routes/dashboard/dashboard.component.ts

```diff
@@ -48,7 +49,8 @@ export class DashboardComponent implements AfterViewInit {
     private activeRoute: ActivatedRoute,
     private loadingService: LoadingService,
     private fb: FormBuilder,
-    private store: Store
+    private store: Store,
+    private urlService: UrlService
```

Comment:
urlService is not needed here. It is mostly created for back navigation (button to navigate to the previous page depending on what page it was).